build/docker/dockerfile.tmpl

```diff
@@ -0,0 +1,71 @@
+
+#----------------------------------------------------------------------------------------------
+FROM redisfab/redis:{{REDIS_VERSION}}-{{ARCH}}-{{OSNICK}} AS redis
+FROM {{OS}} AS builder
+
+SHELL ["/bin/bash", "-l", "-c"]
+
+RUN echo "Building for {{OSNICK}} ({{OS}}) for {{ARCH}} [with Redis {{REDIS_VERSION}}]"
+
+WORKDIR /build
+COPY --from=redis /usr/local/ /usr/local/
+
+ADD . /build
+
+RUN ./deps/readies/bin/getupdates
+RUN ./deps/readies/bin/getpy2
+RUN ./sbin/system-setup.py
+
+RUN /usr/local/bin/redis-server --version
+RUN make fetch
+
+RUN make build SHOW=1
+
+{% if PACK == "1" %}
+RUN make build LITE=1 SHOW=1
```

Comment:
Can these two be combined to make build LITE=1 SHOW=1 COORD=rlec?

Reply:
Not likely. Those are two variations of the same build. I can maybe offer `make ALL=1`?

Reply:
Agreed